lib/core/src/Cardano/Wallet/Primitive/AddressDerivation/Byron.hs

```diff
@@ -266,6 +269,21 @@ hdPassphrase masterKey = Passphrase $
     (unXPub masterKey)
     ("address-hashing" :: ByteString)
 
+generateKeyFromMasterKey
+    :: XPrv
+    -> ByronKey 'RootK XPrv
+generateKeyFromMasterKey = unsafeGenerateKeyFromMasterKey ()
+
+unsafeGenerateKeyFromMasterKey
+    :: DerivationPath depth
+    -> XPrv
+    -> ByronKey depth XPrv
+unsafeGenerateKeyFromMasterKey derivationPath masterKey = ByronKey
+    { getKey = masterKey
+    , derivationPath
+    , payloadPassphrase = hdPassphrase (toXPub masterKey)
+    }
```

Comment:
These functions aren't generating any keys though :grimacing: 
I don't think they are necessary to be honest, the calling code could simply be calling `ByronKey` directly, or, maybe we can simply rename the function to `mkByronKey :: XPrv -> ByronKey 'RootK XPrv`

Reply:
yes, fair remarks. `mkByronKey` is better naming

Reply:
done in https://github.com/input-output-hk/cardano-wallet/pull/1454/commits/af27d7004db13e835c679bfa6d78a9692ed7b28d